scripts/addFromConfig.py

```diff
@@ -176,9 +170,9 @@ def addStuff(cfg, options):
 
     # is the satellite in the DB?  If not add it
     try:
-        satellite_id = dbu.getEntry('Satellite', cfg['satellite']['satellite_name']).satellite_id
+        satellite_id = dbu.getSatelliteIDy(cfg['satellite']['satellite_name'])
```

Comment:
There's no such method getSatelliteIDy. I *think* this change exists because `getEntry` doesn't work happily with postgres. It throws a terrible error if you try to search a numerical field with a string, whereas sqlite simply returns no results. getEntry uses this pretty heavily, and it needs to be updated to work with postgres...then I think this change wouldn't be needed.

Reply:
I think I have a typo with "ID" and "IDy".   You are correct about the "string" search. https://github.com/dnadeau-lanl/dbprocessing/blob/postgresql/dbprocessing/DButils.py#L1992

Reply:
Since all we really need here is the satellite ID, this makes more sense anyhow. Otherwise getEntry calls getSatelliteID anyhow, and then uses it for a lookup, of which we only use the ID...